align_system/algorithms/outlines_regression_adm_comparative.py

```diff
@@ -413,21 +443,37 @@ def top_level_choose_action(self,
         # to a single dictionary with values that are a list of samples
         predicted_kdma_values = merge_samples(predictions)
 
-        # Regress best choice
-        if distribution_matching == 'average':
+        # Select best choice
+        
+        # If we have a scalar value target, use average distribution matching
+        if target_kdmas[0].value is not None:
```

Comment:
No changes required at this time, but I'm wondering if it would ever be the case that some of the targets they provide are scalar and some are distribution-based once we get to the multi KDMA stage (not directed at you)

Reply:
Yeah good point, I'll add a comment

Reply:
Added https://github.com/ITM-Kitware/align-system/pull/82/commits/dfadd402dc323ce4116b00fb5f99ae0900841f7b